pom.xml

```diff
@@ -1944,7 +1944,7 @@
                 <plugin>
                     <groupId>org.skife.maven</groupId>
                     <artifactId>really-executable-jar-maven-plugin</artifactId>
-                    <version>1.0.5</version>
+                    <version>2.1.1</version>
```

Comment:
It looks like a major change. What changed?

Reply:
The way zip is made executable. Plugin works the same